### PR TITLE
Fake a log on and give player id

### PR DESCRIPTION
**Pull request by chrisw000**

### CODE REVIEW

File: BigCheese/Api/AccountController.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using BlueCheese.HostedServices.Bingo;
+
+namespace BigCheese.Api
+{
+    [Route("api/[controller]")]
+    public class AccountController : Controller
+    {
+        private const string Cheese = "Manchego";
```
Comment (jackall3n) [inline BigCheese/Api/AccountController.cs]: 🧀

File: BigCheese/Startup.cs [hunk 2]
```diff
@@ -45,6 +48,22 @@ public void ConfigureServices(IServiceCollection services)
...
+            {
+                options.CheckConsentNeeded = context => true;
+                options.MinimumSameSitePolicy = SameSiteMode.None;
+            });
+
+            services.AddDistributedMemoryCache();
+
+            services.AddSession(options =>
+            {
+                options.Cookie.Name = ".tienda.de.quesos";
```
Comment (jackall3n) [inline BigCheese/Startup.cs]: 🏦🧀

**Review by jackall3n — APPROVED** [review]

File: BigCheese/Api/AccountController.cs (hunk 1 above)
Comment (jackall3n) [reply BigCheese/Api/AccountController.cs]: Adequate cheese

**Comment by chrisw000** [comment]
See the lobby view and lobby js
You'd now post a username to the server and get back a player ID
The username / player ID are faked as a logon
Session tracks the ID so once logged on you can't change your ID / username
The username becomes unique - returns a 403 if the name is already taken
Start / Join games don't do anything if the username / playerID dont match

**Pull request merged**